ecosystem/resources/members/mqt-qao.toml

```
@@ -0,0 +1,12 @@
+name = "MQT QAO"
+url = "https://github.com/cda-tum/mqt-qao"
+description = "MQT Quantum Auto Optimizer translates optimization problems into quantum-compliant formats and solves them with major quantum solvers."
```

```suggestion
description = "Translate optimization problems into quantum-compliant formats and solve them with major quantum solvers."
```

Just removed the package name (will appear above the description) and tweaks the language to be more actionable.